src/ExpressiveInstaller/config.php

```diff
@@ -11,13 +11,22 @@
         'zendframework/zend-expressive-twigrenderer'     => '^1.2.1',
         'zendframework/zend-expressive-zendrouter'       => '^2.0',
         'zendframework/zend-expressive-zendviewrenderer' => '^1.2.1',
-        'zendframework/zend-servicemanager'              => '^2.7.3 || ^3.0',
+        'zendframework/zend-servicemanager'              => '^2.7.3 || ^3.2',
     ],
 
     'require-dev' => [
         'filp/whoops'
     ],
 
+    'application' => [
+        'flat' => [
+            '/Resources/src/ConfigProvider.flat.php' => '/src/App/ConfigProvider.php',
+        ],
+        'modular' => [
+            '/Resources/src/ConfigProvider.modular.php' => '/src/App/src/ConfigProvider.php',
```

Comment:
The problem with that is that we only want *one* of those files copied in — either the flat or the modular version becomes the `ConfigProvider.php` file, and the target is based on the install type. (The two source files are *different*, as they present different template path configuration.)

I'll see if I can come up with another solution, though.